lib/sda/tree.go

```diff
@@ -258,7 +259,7 @@ func (tm TreeMarshal) MakeTree(il *EntityList) (*Tree, error) {
 
 // MakeTreeFromList creates a sub-tree given an EntityList
 func (tm *TreeMarshal) MakeTreeFromList(parent *TreeNode, il *EntityList) *TreeNode {
-	idx, ent := il.Search(tm.EntityId)
+	idx, ent := il.SearchEntity(tm.EntityId)
```

Comment:
I guess at this point, I could say that naming its variable `il` is not a very good name and should be named more `entList` or (my favorite) `entityList`. So you can have `entityList.Search(...)`.
Anyway, that was my 2 cent, I won't push for it so as you like.